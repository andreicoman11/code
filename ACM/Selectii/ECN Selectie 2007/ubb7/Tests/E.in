13
1
7
2
12
6
10
3
8
11
13
4
5
9
