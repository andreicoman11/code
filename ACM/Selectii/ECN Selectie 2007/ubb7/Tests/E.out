1
5 1 3 4 2 6 7
2 1
7 1 4 9 2 11 10 8 3 6 5 12
4 1 6 3 2 5
9 1 8 5 2 4 7 6 3 10
3 1 2
3 1 7 5 2 6 8 4
5 1 6 4 2 10 11 7 3 8 9
4 1 13 11 2 10 6 7 3 5 12 9 8
2 1 4 3
3 1 4 5 2
7 1 8 6 2 9 4 5 3
